test/tests/userobjects/threaded_general_user_object/tests

```diff
@@ -6,6 +6,7 @@
     min_threads = 1
     max_threads = 1
     csvdiff = 'th1.csv'
+    tbb = true
```

Comment:
I don't think we have any civet targets that configure libmesh with `--with-thread-model=tbb`, so these tests will always get skipped. The default thread model is `openmp`. The min clang target uses `pthread`.

Reply:
Yeah. I will need to improve test harness so I can specify the threading model - similarly what we do for compiler, etc.